src/ontology/mondo-edit.obo

```diff
@@ -438962,6 +438965,74 @@ is_a: MONDO:0100355 {source="https://orcid.org/0000-0001-5208-3432"} ! classic o
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3221 xsd:string
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100358
+name: lip herpes
+def: "Any herpes simplex infection that involves the lip." [MONDO:patterns/location]
+synonym: "herpes infectious disease of lip" EXACT [MONDO:patterns/location]
+synonym: "lip herpes infectious disease" EXACT [MONDO:patterns/location]
+is_a: MONDO:0004609 {source="https://orcid.org/0000-0001-5208-3432"} ! herpes simplex infectious disease
+intersection_of: MONDO:0004609 ! herpes simplex infectious disease
+intersection_of: disease_has_location UBERON:0001833 ! lip
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3348 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100359
+name: herpes simplex type 1 infectious disease
+def: "A disease caused by infection with herpes simplex type 1." [MONDO:patterns/infectious_disease_by_agent]
+is_a: MONDO:0004609 {source="https://orcid.org/0000-0001-5208-3432"} ! herpes simplex infectious disease
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3348 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100360
+name: herpes simplex type 2 infectious disease
+def: "A disease caused by infection with herpes simplex type 2." [MONDO:patterns/infectious_disease_by_agent]
+is_a: MONDO:0004609 {source="https://orcid.org/0000-0001-5208-3432"} ! herpes simplex infectious disease
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3348 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
```

Comment:
same question as above about the logical definition including herpes simplex type 2

Reply:
yep, I'll add it as soon as I get the NCBI taxon terms imported

Reply:
done